lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwift.cpp

```diff
@@ -101,3 +101,16 @@ bool TypeSystemSwift::ShouldTreatScalarValueAsAddress(
   return Flags(GetTypeInfo(type, nullptr))
       .AnySet(eTypeInstanceIsPointer | eTypeIsReference);
 }
+
+/// Get the index of the child of "clang_type" whose name matches. This function
+/// doesn't descend into the children, but only looks one level deep and name
+/// matches can include base class names.
```

Comment:
nit: According to the LLVM style, Doxygen comments _only_ go into the header file.

Reply:
Meaning this should use only `//`? Or that this documentation should be deleted since it more or less reproduces the documentation in the header?

Reply:
I went with the latter, deleting this comment.